lib/FrontendTool/FrontendTool.cpp

```diff
@@ -237,12 +237,156 @@ template <> struct ObjectTraits<LoadedModuleTraceFormat> {
 }
 }
 
+namespace {
```

Comment:
Instead of the anonymous namespace, we typically use `static` for helper functions. See the [LLVM Coding Standards](https://www.llvm.org/docs/CodingStandards.html#anonymous-namespaces).